app/views/presentations/show.html.erb

```diff
@@ -10,3 +10,9 @@
   <dt>Date</dt>
   <dd><%= @presentation.event.date %></dd>
 </dl>
+
+<%= link_to 'Edit', edit_presentation_path(@presentation) %>
+<%= link_to 'Delete', presentation_path(@presentation),
+  method: :delete,
+  data: { confirm: 'Are you sure?' } %>
```

Comment:
As discussed, rubocop only checks the .rb files so this file wouldn't have been been checked. However you bring up an excellent point - we seem to be using different indenting styles in our views to break up long lines. I just found this thoughtbot styleguide that has an opinion on this:
https://github.com/thoughtbot/guides/blob/master/style/erb/sample.html.erb